Source/SwiftLintFrameworkTests/ASTRuleTests.swift

```diff
@@ -97,6 +97,10 @@ class ASTRuleTests: XCTestCase {
         }
     }
 
+    func testPermittedCapitalizedVariableNames() {
```

Comment:
This test is unnecessary because the rule's `nonTriggeringExamples` are tested automatically.